apps/root/src/pages/aggregator/swap-container/components/step1/index.tsx

```diff
@@ -176,11 +163,9 @@ const SwapFirstStep = React.forwardRef<HTMLDivElement, SwapFirstStepProps>((prop
             balance={balance}
             maxBalanceBtn
           />
-        </StyledContentContainer>
-        <ToggleButton isLoadingRoute={isLoadingRoute} />
-      </Grid>
-      <Grid item xs={12} sx={{ paddingTop: '8px !important' }}>
-        <StyledContentContainer>
+          <ToggleButton isLoadingRoute={isLoadingRoute} />
+        </Grid>
+        <Grid item xs={12} sx={{ paddingTop: '8px !important' }}>
```

Comment:
is this padding necessary now?

Reply:
I left this styling as it was, because it keeps the token picker separated as it should be. I agree we should remove it and implement our new theme, but maybe we can address that when we style this component on BLY-1637